performance-tests/perf-test-commits.sh

```diff
@@ -0,0 +1,86 @@
+#! /usr/bin/env bash
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+PERF_TESTS_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+source "${PERF_TESTS_DIR}"/../scripts/common.sh
+
+GREEN='\033[0;32m'
+NOCOLOR='\033[0m'
+
+FIND_COMMAND="find"
+if [ "$OS" = 'Darwin' ] && resolveCommand gfind ; then
+  FIND_COMMAND=gfind
+  ENABLE_REGEX="-regextype posix-extended"
+elif [ "$OS" = 'Darwin' ] ; then
+  # for MacOS
+  ENABLE_REGEX="-E"
+else
+  # for Linux and Windows
+  ENABLE_REGEX="-regextype posix-extended"
+fi
+
+COMMITS=( "$@" )
+
+#Cross platform temp directory creation based on https://unix.stackexchange.com/a/84980
+RESULTS_DIR=${RESULTS_DIR:=$(mktemp -d -t kroxyliciousPerfTestResults.XXXXXX 2>/dev/null || mktemp -d -t 'kroxyliciousPerfTestResults')}
+JSON_TEMP_DIR=${JSON_TEMP_DIR:=$(mktemp -d -t kroxyliciousPerfTestJson.XXXXXX 2>/dev/null || mktemp -d -t 'kroxyliciousPerfTestJson')}
```

Comment:
`mktemp` on Mac OS automatically includes the `XXXXX` portion hence the simpler pattern.